src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -2867,3 +2867,29 @@ def test_python_unhandled_exception(self):
     self._validate_get_crash_data(data, expected_type, expected_address,
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
+                                  
+  def test_gdb_sigtrap(self):
+    """Test for GDB stack."""
+    data = self._read_test_data(
+        'gdb_sigtrap.txt')
+    expected_type = 'SIGTRAP'
+    expected_address = '0x000000000ac8'
+    expected_state = ('__aeabi_memcpy\nxymodem_trnasfer\nLoadImageFromUsb30\n')
+    expected_stacktrace = data
+    expected_security_flag = True
```

Comment:
This seems incorrect since the crash address is near null, expected security flag should be False. I think this is because of needing to add that signature in
https://github.com/google/clusterfuzz/blob/191ee1fce1351c0016abbae1186d0792de83aeae/src/python/crash_analysis/crash_analyzer.py#L38 and then it will come out as False.

Reply:
We discussed we will do this in a separate CL because these addresses will be near null but they may be security crashes still.